packages/analyzer/src/features/analyse-phase/creators/createClass.js

```diff
@@ -24,7 +24,8 @@ export function createClass(node, moduleDoc, context) {
     slots: [],
     members: [],
     events: [],
-    attributes: []
+    attributes: [],
+    customStates: [],
```

Comment:
This is not standardized in the schema, right? If not we should add it there first

Reply:
Correct. In fact turns out @luke_warlow has a PR for it here:

https://github.com/webcomponents/custom-elements-manifest/pull/128

Looks like they went with:

`cssStates` so I'm guessing we should do the same. It also hasn't been merged or reviewed.

Reply:
Lets go with whats suggested in the schema, and lets re-use that for the jsdoc tag

I left a comment on the PR on the schema and pinged Justin to ask him to also look at it

Reply:
Iirc I went with cssStates because I want to use the manifest for defining built-in elements too. But I'm not super attached to the naming.